### PR TITLE
[3.x] Various updates for next alpha

### PR DESCRIPTION
**Pull request by haringsrob**
fixes https://github.com/area17/twill/issues/1761
fixes https://github.com/area17/twill/issues/1795
fixes https://github.com/area17/twill/issues/1811
fixes https://github.com/area17/twill/issues/1781

Fixes issue that would not render the default text(wysiwyg) block.

### CODE REVIEW

**Comment by haringsrob** [comment]
@ifox as I did change a bit of vue, can you do a quick review? This pr contains all planned for the next alpha and should resolve all bugs we discussed.

File: frontend/js/components/table/tableCell/generic/TableCellName.vue [hunk 1]
```diff
@@ -1,30 +0,0 @@
-<template>
-  <a :href="editUrl"
-     v-if="!row.hasOwnProperty('deleted') && (row.edit)"
```
Comment (ifox) [inline frontend/js/components/table/tableCell/generic/TableCellName.vue:3]: Is this condition respected by the new table builder? When we link to edit, we should check if the record is not soft deleted.

File: frontend/js/components/table/tableCell/generic/TableCellName.vue [hunk 2]
```diff
@@ -1,30 +0,0 @@
...
-     class="tablecell__name"
-     @click="preventEditInPlace($event)">
```
Comment (ifox) [inline frontend/js/components/table/tableCell/generic/TableCellName.vue:5]: Is the `editInModal` index option still functional after this change?

Comment (haringsrob) [reply frontend/js/components/table/tableCell/generic/TableCellName.vue:5]: It was not but the last commit takes care of this.

File: frontend/js/components/table/tableCell/generic/TableCellName.vue (hunk 1 above)
Comment (haringsrob) [reply frontend/js/components/table/tableCell/generic/TableCellName.vue:3]: Was not taken care of but it is now!

**Pull request merged**